### PR TITLE
Reduce polling

### LINKED ISSUE #2451: Reduce polling based on known state
We can reduce overall traffic by ensuring that

- Feed polls don't fire if they already known there is unread
- Notifications poll fires less frequently if there are is unread (1/2 as often)

### LINKED ISSUE #2454: Change preferences polling into on-foreground fetches
Currently we poll for preferences regularly. I believe we can switch to a fetch on foreground events (web and mobile) and then also fetch when viewing preferences directly and that should cover it.

### PR DESCRIPTION
**Pull request by pfrazee**
- a4c51da869f321a9f39f2da253e76566bb6f4450 Stops fetching preferences and profiles on an interval and instead does it on focus (close #2454)
- 7f9827674b1f8900457ba5f9b925b2a0926fbc56 Makes it possible to refresh the saved feeds in the feeds screen
- 2963d5789c1a7f02f7227636c1264465e6d982f2 and b7686080c9868777feb79daece148d8777412b32 Reduces feed and notifications polling if there's known notifications (close #2451)
- 4d87706c3513edc58c0200d3849eb0b5d723f2ca Disables the hard refresh after 1 hour to test if that's the cause of the random reload on start bug

### CODE REVIEW

File: src/view/com/posts/Feed.tsx [hunk 1]
```diff
@@ -146,11 +149,12 @@ let Feed = ({
   React.useEffect(() => {
     if (enabled) {
       const timeSinceFirstLoad = Date.now() - lastFetchRef.current
-      if (timeSinceFirstLoad > REFRESH_AFTER) {
+      // DISABLED need to check if this is causing random feed refreshes -prf
+      /*if (timeSinceFirstLoad > REFRESH_AFTER) {
```
Comment (estrattonbailey) [inline src/view/com/posts/Feed.tsx:153]: Ooooooooh yeah this looks promising

File: src/view/screens/Feeds.tsx [hunk 2]
```diff
@@ -151,9 +152,12 @@ export function FeedsScreen(_props: Props) {
   }, [query, debouncedSearch])
   const onPullToRefresh = React.useCallback(async () => {
     setIsPTR(true)
-    await refetchPopularFeeds()
+    await Promise.all([
+      refetchPreferences().catch(_e => undefined),
+      refetchPopularFeeds().catch(_e => undefined),
```
Comment (estrattonbailey) [inline src/view/screens/Feeds.tsx:157]: Do we need to wait for prefs before fetching other feeds or nah

File: src/lib/react-query.ts [hunk 3]
```diff
@@ -1,11 +1,39 @@
-import {QueryClient} from '@tanstack/react-query'
+import {AppState, AppStateStatus} from 'react-native'
+import {QueryClient, focusManager} from '@tanstack/react-query'
+import {isNative} from '#/platform/detection'
+
+focusManager.setEventListener(onFocus => {
```
Comment (estrattonbailey) [inline src/lib/react-query.ts:5]: TIL

**Review by estrattonbailey — APPROVED** [review]

File: src/view/screens/Feeds.tsx (hunk 2 above)
Comment (pfrazee) [reply src/view/screens/Feeds.tsx:157]: nah

**Pull request merged**